src/doc/grid.h

```diff
@@ -75,6 +76,9 @@ namespace doc {
     // Returns an array of tile positions that are touching the given region in the canvas
     std::vector<gfx::Point> tilesInCanvasRegion(const gfx::Region& rgn) const;
 
+    // Returns a mask aligned to the current grid, starting from other not aligned mask
+    Mask makeAlignedMask(const Mask* mask) const;
```

Comment:
This function seems more related to masks than grids (e.g. a `Mask` constructor). There are two options: 1) move this function as a member function of the Mask class, or 2) create an utility function that receives both the mask and the grid as parameters (see `doc/util.h`).